indra/llprimitive/llgltfloader.h

```diff
@@ -79,6 +79,7 @@ class gltf_render_material
     double      normalScale;    // scale applies only to X,Y components of normal
     double      occlusionScale; // strength multiplier for occlusion
     LLColor4    emissiveColor;  // emissive mulitiplier, assumed linear encoding (spec 2.0 is silent)
+    double      emissiveStrength;  // multiplier for emissiveColor
```

Comment:
this is confusing to me that it is `double` here but handled as `F32` everywhere else, but seems like that is what we do for all the other fields so ok i guess

Reply:
Yeah, I was a bit weirded out by that.  I'm keeping with the established convention more or less.

Reply:
Looking at the GLTF spec, it does seem there are cases where single precision floating point is enforced. From the section on "Accessors Bounds":

> For floating-point components, JSON-stored minimum and maximum values represent single precision floats and SHOULD be rounded to single precision before usage to avoid any potential boundary mismatches.

Not relevant to this PR, but might be worth keeping in mind for the future. cc: @RunitaiLinden